docs/Language/C++/C++.md

```diff
@@ -9,9 +9,135 @@ has_children: true
 {{ page.title }}
 ======================
 
-{% include under_construction.html %}
+C++ is a general-purpose programming language that was developed as an extension of the C programming language. It was created by Bjarne Stroustrup at Bell Labs in the early 1980s. C++ introduces object-oriented programming features to C, allowing developers to write more organized and modular code.
 
+-----------------------------------------------------------------------
 
-<br>
+### Official Documentation
 
-<br>
\ No newline at end of file
+- Microsoft Documentation: https://learn.microsoft.com/en-us/cpp/cpp/
+- CPlusPlus: https://cplusplus.com/
+- CPP Reference: https://en.cppreference.com/w/
+- IsoCPP: https://isocpp.org/std/the-standard
+
+-----------------------------------------------------------------------
+
+### Features
+
+- Object-Oriented Programming (OOP): C++ supports the four main principles of OOP: encapsulation, inheritance, polymorphism, and abstraction. This makes it possible to model real-world entities more effectively.
```

Comment:
Too wordy, consider "Re-tuning" it.   And I believe that rest of the content should be ordered based on  " encapsulation, inheritance, polymorphism, and abstraction " Ordering given.  Personally I would tune it to Talk about what the Terminology means And then what falls into that category.   " Object... Or known as (OOP) " <- Or wise versa. some text ... And then categories like you already listed.

Reply:
There is a dedicated OOP section in "Concepts/OOP"